source/results/flux_points/index.rst

```diff
@@ -120,8 +145,9 @@ SED format defines an *SED Type* string which is set with the
 columns that must be present in the SED.  The SED types and their
 required columns are given in the following list:
 
-* ``diff_flux_points``: ``e_ref``, ``dnde``, ``dnde_err``
-* ``int_flux_points``: ``e_min``, ``e_max``, ``flux``, and ``flux_err``
+* ``dnde``: ``e_ref``, ``dnde``
+* ``flux``: ``e_min``, ``e_max``, ``flux``
+* ``eflux``: ``e_min``, ``e_max``, ``eflux``
```

Comment:
+1 -- I like this "minimal" definition of SED types, not mentioning error or UL fields.